server/sample.config.json

```diff
@@ -9,5 +9,10 @@
         "username": "testuser",
         "password": "testpass",
         "mission_id": 1
+    },
+    "socket": {
+        "connection": true,
+        "host": "127.0.0.1",
+        "port": 4000
     }
```

Comment:
Instead of calling it `socket`, you may want to embed it in the `uav` section and call it `image` or something.

Reply:
That's a good idea.